statcord/client.py

```diff
@@ -72,18 +86,11 @@ def _format_traceback(e: Exception) -> str:
     def _get_user_count(self) -> int:
```

Comment:
I'm a bit sad since this has to go through tons of guilds,  but whatever

Reply:
You could check if `bot.intents.members and bot.chunk_guilds_at_startup`, though I don't know if that value is stored on the bot class.